openwisp_users/api/mixins.py

```diff
@@ -107,8 +107,6 @@ def get_queryset(self):
 
     def assert_parent_exists(self):
         parent_queryset = self.get_parent_queryset()
-        if not self.request.user.is_superuser:
-            parent_queryset = self.get_organization_queryset(parent_queryset)
```

Comment:
This doesn't look like something that was requested, why are you changing this?

Reply:
This can't be removed as it's intended to restrict non superusers so they can only see objects of their organization and not objects of other organizations. Removing this is absolutely wrong. The fact that no tests are failing for this is bad and requires opening a new issue to add at least one which would fail without this code.